scrapy_webarchive/wacz/wacz_file.py

```diff
@@ -49,11 +50,15 @@ def get_warc_from_cdxj_record(self, cdxj_record: CdxjRecord) -> Union[WARCRecord
         warc_part = BytesIO(file_part)
         return WARCReader(warc_part).read_record()
 
-    def get_warc_from_url(self, url: str) -> Union[WARCRecord, None]:
+    def get_warc_from_url(self, url: str) -> Tuple[Union[WARCRecord, None], Union[CdxjRecord, None]]:
```

Comment:
it seems that we either have a `WARCRecord` and ` CdxjRecord`, or neither. So wouldn't the return type be
```
Union[Tuple[WARCRecord, CdxjRecord], Tuple[None, None]]
```